src/main/java/edu/ucsb/cs156/happiercows/jobs/UpdateCowHealthJob.java

```diff
@@ -0,0 +1,11 @@
+package edu.ucsb.cs156.happiercows.jobs;
```

Comment:
Same comment as above.  Needs a test similar to:

https://github.com/ucsb-cs156-f22/f22-6pm-happycows/blob/main/src/test/java/edu/ucsb/cs156/happiercows/jobs/MilkTheCowsJobTests.java